static/js/interactive-map/location-card/add-card-listeners.js

```diff
@@ -0,0 +1,41 @@
+const getIndexOfThisCard = () => {
```

Comment:
I think we can have this file export a single function that registers listeners, and have it be accessed through HitchhikerJS instead of a partial? Otherwise this code is added to the bundle multiple times, once per card. 
In general mixing hbs and js is a headache (I know this was how it was done before this pr but this seems like the time to change it)

Reply:
Sure, I can definitely see how mixing JS and HBS can make things more confusing

Reply:
Added this to the locator bundle rather than HitchhikerJS